omod-common/src/main/java/org/openmrs/module/webservices/rest/web/resource/impl/BasePageableResult.java

```diff
@@ -58,6 +67,11 @@ public SimpleObject toSimpleObject(Converter preferredConverter) throws Response
 				links.add(context.getPreviousLink());
 			ret.add("links", links);
 		}
+		if (context.getRequest() != null) {

```

Comment:
Does context.getParameter throw an exception if context.getRequest is null? If yes, then context.getParameter should be fixed to return null instead...